opensearch-operator/pkg/reconcilers/util/util.go

```diff
@@ -271,3 +268,45 @@ func GetSha1Sum(data []byte) (string, error) {
 
 	return hex.EncodeToString(hasher.Sum(nil)), nil
 }
+
+// GetClusterHealth returns the health of OpenSearch cluster
+func GetClusterHealth(ctx context.Context, k8sClient client.Client, cluster *opsterv1.OpenSearchCluster) opsterv1.OpenSearchHealth {
+	osClient, err := CreateClientForCluster(ctx, k8sClient, cluster, nil)
+
+	if err != nil {
+		return opsterv1.OpenSearchUnknownHealth
```

Comment:
The error should be logged at least, for debugging purposes. For the other places also.

Reply:
I didn't want to log it because when the cluster is first coming up, it will always fail to create the client and the logs are filled with error messages. Same at other places as well, when the sts is just being created, it returns a few not found error

Reply:
Maybe log it on debug level, then it should normally not appear as the default log level is info. And in case someone has a problem we can ask for debug logs.